.github/workflows/release-swift-toolchain-binary-sizes.yml

```diff
@@ -0,0 +1,150 @@
+        # configure-aws-credentials v4.0.1 release
+        uses: thebrowsercompany/gha-aws-ssm-get-parameter@v1
+        with:
+          aws-role-to-assume: ${{ secrets.SWIFT_TOOLCHAIN_UPLOADER_ROLE_ARN }}
+          aws-role-session-name: SwiftToolchainMetricsUploader
+          aws-ssm-parameter: "/shared/secrets/GITHUB_ACTIONS_BQ_DATA_UPLOAD"
+          save-to-filepath: ${{ github.workspace }}/.google_application_credentials
+
+      - name: Setup Google application default credentials
+        run: echo "GOOGLE_APPLICATION_CREDENTIALS=${{ github.workspace }}/.google_application_credentials" | Out-File -FilePath $env:GITHUB_ENV -Append
+
+      - name: Install Swift toolchain
+        uses: compnerd/gha-setup-swift@main
+        with:
+          github-repo: thebrowsercompany/swift-build
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          release-asset-name: installer-${{ matrix.toolchain_arch }}.exe
+          release-tag-name: ${{ env.SWIFT_TOOLCHAIN_VERSION }}
+
+      - name: Store Swift Toolchain root in environment variable
+        run: |
+          # Locate the toolchain installation <root> by walking up from
+          # '<root>/Toolchains/<toolchain-version>/usr/bin/swift.exe'.
+          $SwiftPath=$(Get-Command swift).Source
+          $SwiftInstallRoot=$(Resolve-Path -LiteralPath $SwiftPath\..\..\..\..\..)
+          echo "SWIFT_INSTALL_ROOT=${SwiftInstallRoot}" | Out-File -FilePath $env:GITHUB_ENV -Append
+
+      # For scripts/python/binary_sizes
+      - name: Install Python dependencies
+        run: |
+          pip install google-cloud-bigquery
+          pip install google-auth-oathlib
+          pip install pandas
+
+      # For the options file schema, see https://github.com/google/bloaty/blob/main/src/bloaty.proto.
+      - name: Generate Bloaty options file
+        run: |
+          New-Item -Path ${{ env.BLOATY_OPTIONS_FILE }}
+
+          # Generate the list of input files.
+          # Each input filename must be an absolute path.
+          # Unix-style path separators are required.
+          # The textproto output must be ascii encoded.
+          Get-ChildItem -Recurse -Name -Path ${{ env.SWIFT_INSTALL_ROOT }} -Include "*.dll","*.exe" | `
+            %{ echo "filename: `"${{ env.SWIFT_INSTALL_ROOT }}\$_`"" } | `
+            %{ $_ -replace '\\','/' } | `
+            Out-File ${{ env.BLOATY_OPTIONS_FILE }} -Append -Encoding ascii
+
+          echo "sort_by: SORTBY_FILESIZE" | Out-File ${{ env.BLOATY_OPTIONS_FILE }} -Append -Encoding ascii
+
+          # At the time of witing, these are the only data sources supported by
+          # bloaty's PE/COFF format.
+          echo "data_source: `"inputfiles`"" | Out-File ${{ env.BLOATY_OPTIONS_FILE }} -Append -Encoding ascii
+          echo "data_source: `"segments`"" | Out-File ${{ env.BLOATY_OPTIONS_FILE }} -Append -Encoding ascii
+
+      - name: Show bloaty options file
+        run: Get-Content -Path ${{ env.BLOATY_OPTIONS_FILE }}
```

Comment:
I wonder if we can sink this logic into `gha-google-bloaty`. We could take a pattern of paths to measure and generate the configuration setup and an option to display the configuration.

Reply:
I need some time to think about how to implement this. Let's do it in a follow up.  After trying it out, the API is awkward because Powershell's `Get-ChildItem -Path` doesn't support recursive glob patterns - you have to do `Get-ChildItem -Path ... -Include ...` or `Get-ChildItem -Path ... -Filter ...`  and both involve passing two arguments. I think it's OK if we don't have this yet.

Reply:
`Get-ChildItem -Recurse` will do the recursive bit :)

Reply:
Yeah, let's do this later. I need to time to debug and test the behavior and I don't think it's critical for this workflow.